Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,10 +1,58 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using BootCamp.Chapter.Items;
+
+namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
         {
+            Player playerOne = new Player("Arthas Menethil", 50, 10, new Inventory(), new Equipment());
+            EquipItems_And_DisplayStatistics(playerOne);
+        }
+
+        // This code is ugly but it properly displays the stats of the player after items are equipped.
+        private static void EquipItems_And_DisplayStatistics(Player player)
+        {
```

Comment:
This is real ugly code there, but I've wanted a quick way of testing whether equipping items change stats.